cmd/version.go

```diff
@@ -45,20 +45,30 @@ type version struct {
 	GoArch             string `json:"goArch"`
 }
 
-// versionString returns the CLI version
+// versionString returns the Full CLI version
 func versionString() string {
-	if kubeBuilderVersion == unknown {
+	if cliVersion == unknown {
 		if info, ok := debug.ReadBuildInfo(); ok && info.Main.Version != "" {
-			kubeBuilderVersion = info.Main.Version
+			cliVersion = info.Main.Version
```

Comment:
I think here we should keep it as before using `kubeBuilderVersion`
It is not used for external projects.
Only in the places that are imported and used for external projects we need use generic terms and names